scenarios/basics/entity_resolution/SPECIFICATION.md

```diff
@@ -0,0 +1,289 @@
+c
+Continuing with the program...
+
+Job ID: ec2dbd1717624b2b806ed93a04c20049
+The matching job was successfully started.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+4. Get details for job ec2dbd1717624b2b806ed93a04c20049
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Job status: QUEUED
+Job details: GetMatchingJobResponse(JobId=ec2dbd1717624b2b806ed93a04c20049, StartTime=2025-01-30T18:37:57.475Z, Status=QUEUED)
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+5. Get Schema Mapping.
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+Attribute Name: id, Attribute Type: UNIQUE_ID
+Attribute Name: name, Attribute Type: STRING
+Attribute Name: email, Attribute Type: STRING
+Schema mapping retrieval completed.
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+6. List Schema Mappings.
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+7. Tag the schema450resource.
+Tags can help you organize and categorize your Entity Resolution resources.
+You can also use them to scope user permissions by granting a user permission
+to access or change only resources with certain tag values.
+In Entity Resolution, SchemaMapping and MatchingWorkflow can be tagged. For this example,
+the SchemaMapping is tagged.
+
+Successfully tagged the resource.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+8. Delete the AWS Entity Resolution Workflow.
+You cannot delete a workflow that is in a running state.
+Would you like to wait for the workflow to complete.
+This can take up to 30 mins (y/n).
+
+n
+
+Enter 'c' followed by <ENTER> to continue:
+c
+Continuing with the program...
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+This concludes the AWS Entity Resolution scenario.
+--------------------------------------------------------------------------------
+
+
+```
+
+## SOS Tags
+
+The following table describes the metadata used in this Basics Scenario.
+
+| action                  | metadata file          | metadata key                  |
+|-------------------------|------------------------|-------------------------------|
+| `createWorkflow`        | entity_metadata.yaml   | entity_CreateWorkflow         |
```

Comment:
All of these keys and the filename are incorrect, please make sure they match the keys and filename in the metadata in this PR. Those keys all start with `entityresolution`.

Reply:
fixed